cluster/rbac/manager.go

```diff
@@ -25,15 +25,20 @@ import (
 var ErrBadRequest = errors.New("bad request")
 
 type Manager struct {
-	authZ  authorization.Controller
-	logger logrus.FieldLogger
+	enabled bool
+	authZ   authorization.Controller
+	logger  logrus.FieldLogger
 }
 
-func NewManager(authZ authorization.Controller, logger logrus.FieldLogger) *Manager {
-	return &Manager{authZ: authZ, logger: logger}
+func NewManager(controllerEnabled bool, authZ authorization.Controller, logger logrus.FieldLogger) *Manager {
+	return &Manager{enabled: controllerEnabled, authZ: authZ, logger: logger}
```

Comment:
here we could just do `enabled: authZ != nil` and wouldn't have to add another entry to the big config?

Reply:
the thought behind that it shall do something if it's expected to be enabled but wasn't initialized

Reply:
it's better to handle it with error though but that will require changes to the whole up stream callers